### DISCUSSION TITLE
option to create an account without needing an email

### DISCUSSION THREAD

**Discussion by pinkjimrussiagirl** [stoatchat/discussions#619] [Feature Suggestions]
i have a friend that lives in the same country as me that doesn’t have an email and can’t join revolt because his parents told him he isn’t allowed an email address yet, so making an option to create accounts without needing an email will be useful for people who don’t have an email address or aren’t able to access an email address

**Comment by Rexogamer** [comment]
This would be prone to a lot of abuse - in addition, in your specific case, unless their parents are particularly strict I doubt they're old enough to use the platform?

**Reply by pinkjimrussiagirl** [reply]
what sort of abuse would it cause, also the friend is 13 years old, also he does have an email but has parental controls on by his parents (my parents also put parental controls on my stuff but isn’t as strict as my friend’s parents)

**Reply by Rexogamer** [reply]
Removed off-topic material.

Requiring emails makes it much harder for spammers and trolls to sign up - if you didn't need an email, you could create thousands of accounts and flood the platform very easily. While it is unfortunate that your friend's parents are being restrictive like this, that is outside our control and ultimately their decision.

**Reply by pinkjimrussiagirl** [reply]
why did you edit my reply

also did you find someone that can change the status icons yet or not

**Reply by Rexogamer** [reply]
The explanation was the first sentence. ***Stop repeatedly asking about unrelated topics.***

**Reply by pinkjimrussiagirl** [reply]
ok fine but idk where i can post about how to change status icons using css

**Reply by Rexogamer** [reply]
You have already made a post about this. Stop bringing it up in unrelated issues.

**Reply by pinkjimrussiagirl** [reply]
ok i will stop and post in the correct place

**also this discussion is now closed since i didn’t get much help from it**
